scripts/fast-reboot

```diff
@@ -416,6 +416,10 @@ function setup_reboot_variables()
         local fstype=$(blkid -o value -s TYPE ${sonic_dev})
         BOOT_OPTIONS="${BOOT_OPTIONS} ssd-upgrader-part=${sonic_dev},${fstype}"
     fi
+
+    # Set governor to performance to speed up boot process.
+    # The governor is reset back to kernel default in warmboot-finalizer script.
+    BOOT_OPTIONS="${BOOT_OPTIONS} cpufreq.default_governor=performance"
```

Comment:
I suggest that you don't add this as a default behavior. This needs to be set by an option provided to warm/fast-reboot script.

That way we can tightly control when/if this is used.

Reply:
@vaibhavhd 

I don't think an optimization is an optinal feature.

We already did optimizations as default behaviour, e.g. LACP retry count, that even had a larger impact as the changes are applied also on the peer side.

Also, from the testing perspective we would need to test all combinations of possible optimization flags user may pass to the script.

From the user perspective, it becomes unclear what flags he needs to use, what is heavily tested and what is not.

Reply:
Does this need an architecture check? Is this supported on armhf and arm64 as well?

Reply:
This may be useful in some scenarios and I think user should be given an option to control this behavior. Some examples I can think of are:
1. High CPU usage during boot up path.
2. High thermal load due to #1.
3. If this mode has dependency on underlying hardware then it depends on the hardware if this is supported.

Mostly these concerns are not critical on new platforms. But some old/slow platforms do not really want this high CPU, high fan activity.

Reply:
@vaibhavhd @saiarcot895 I limit this setting for nvidia where this is tested and working.